src/OrchardCore/OrchardCore.Search.Elasticsearch.Core/Models/ElasticSettings.cs

```diff
@@ -8,6 +8,20 @@ public class ElasticSettings
 
         public static string StandardAnalyzer = "standardanalyzer";
```

Comment:
This should be defaultAnalyzer

Reply:
This is an ElasticSettings class which should contain settings for an Elastic index. Not an enumeration of possible analyzers. To remove any confusion, we could rename StandardAnalyzer to DefaultAnalyzer but I don't think it is necessary.

Reply:
I removed the added constants in this file